inc/saineighbor.h

```diff
@@ -115,6 +115,17 @@ typedef enum _sai_neighbor_entry_attr_t
      */
     SAI_NEIGHBOR_ENTRY_ATTR_COUNTER_ID,
 
+    /**
+     * @brief Remote Out Logical Interface
+     *
+     * Defines the remote neighbor interface when the destination is on a remote line card
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 0
+     */
+    SAI_NEIGHBOR_ENTRY_ATTR_REMOTE_OUTLIF,
+
     /**
```

Comment:
During the review it was agreed that there would be an explicit attribute indicate whether or not the "REMOTE_OUTLIF" attribute is being imposed by the application or to be allocated by SAI. I dont see that attribute here.

Also the REMOTE_OUTLIF was going to be changed to "ENCAP_INDEX" or "ENCAP_ID".

Reply:
Changing to SAI_NEIGHBOR_ENTRY_ATTR_ENCAP_INDEX